sema/discovery/__main__.py

```
@@ -132,16 +132,23 @@ def normalise_mime_type_requests(request_mimes: Iterable | str) -> str:
-    request_mimes = request_mimes.split(",") if request_mimes else []
-    return ",".join({normalise_mime_type_requests(mt) for mt in request_mimes})
+    request_mimes = SemaArgsParser.args_joined(request_mimes)  # type: ignore
+    request_mimes = request_mimes.split(",") if request_mimes else []  # type: ignore # noqa
+    return ",".join({normalise_mime_type_requests(mt) for mt in request_mimes})  # type: ignore # noqa
 
 
 def make_service(args: Namespace) -> Discovery:
     """Make the service with the passed args"""
+
+    if args.url is None and args.url_option is None:
+        raise ValueError("No URL provided")
```

Ruff

Avoid specifying long messages outside the exception class

(TRY003)